src/Stream/StreamFactoryInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Psr\Http\Factory\Stream;
+
+use Psr\Http\Message\StreamInterface;
+
+interface StreamFactoryInterface
```

Comment:
Makes sense to me. There's no way to attach a stream after `StreamInterface` is created, so creating an empty stream is fairly pointless and `ResourceStreamFactoryInterface` covers that case.